react/utils/splunkLogger.ts

```diff
@@ -0,0 +1,16 @@
+import SplunkEvents from 'splunk-events'
+
+const SPLUNK_ENDPOINT = 'https://splunk72-heavyforwarder-public.vtex.com:8088'
+const SPLUNK_TOKEN = 'cce4a8e7-6e7a-40a0-aafb-ac45b0e271ba'
```

Comment:
Is this okay to be public? 😂  I really don't know

Reply:
@victorhmp There's already a (different) Splunk token used on checkout UI that's public so 🤷‍♂️

Reply:
Alright then 😂